src/frontend/Sider/Behandling/Felles/faktiskMålgruppe.ts

```diff
@@ -7,6 +7,52 @@ export enum FaktiskMålgruppe {
     TODO = 'TODO',
 }
 
+const faktiskeMålgrupper: Record<Stønadstype, Record<FaktiskMålgruppe, boolean>> = {
+    [Stønadstype.BARNETILSYN]: {
+        NEDSATT_ARBEIDSEVNE: true,
+        ENSLIG_FORSØRGER: true,
+        GJENLEVENDE: true,
+        TODO: false,
+    },
+    [Stønadstype.LÆREMIDLER]: {
+        NEDSATT_ARBEIDSEVNE: true,
+        ENSLIG_FORSØRGER: true,
+        GJENLEVENDE: true,
+        TODO: false,
+    },
+    [Stønadstype.BOUTGIFTER]: {
+        NEDSATT_ARBEIDSEVNE: true,
+        ENSLIG_FORSØRGER: true,
+        GJENLEVENDE: true,
+        TODO: false,
+    },
+    [Stønadstype.DAGLIG_REISE_TSO]: {
+        NEDSATT_ARBEIDSEVNE: true,
+        ENSLIG_FORSØRGER: true,
+        GJENLEVENDE: true,
+        TODO: false,
+    },
+
+    [Stønadstype.DAGLIG_REISE_TSR]: {
+        NEDSATT_ARBEIDSEVNE: false,
+        ENSLIG_FORSØRGER: false,
+        GJENLEVENDE: false,
+        TODO: true,
+    },
+};
+
+export const faktiskeMålgrupperForStønad: Record<Stønadstype, FaktiskMålgruppe[]> = Object.entries(
+    faktiskeMålgrupper
+).reduce(
+    (prev, [stønadstype, faktiskMålgruppe]) => {
+        prev[stønadstype as Stønadstype] = Object.entries(faktiskMålgruppe)
+            .filter(([, skalMed]) => skalMed)
+            .map(([målgruppe]) => målgruppe) as FaktiskMålgruppe[];
+        return prev;
+    },
+    {} as Record<Stønadstype, FaktiskMålgruppe[]>
+);
```

Comment:
Et alternativ til linje 10-54 er å definere opp ting litt mer manuelt, som gjør at ting blir en smule mer eksplisitt og lesbart. 

Jeg tenker at både din løsning og dette alternativet funker fint, så det er litt opp til deg hva du foretrekker 😌

```suggestion
const MÅLGRUPPER_NAY = [
    FaktiskMålgruppe.NEDSATT_ARBEIDSEVNE,
    FaktiskMålgruppe.ENSLIG_FORSØRGER,
    FaktiskMålgruppe.GJENLEVENDE,
] as const;

export const faktiskeMålgrupperForStønad: Record<Stønadstype, FaktiskMålgruppe[]> = {
    [Stønadstype.BARNETILSYN]: [...MÅLGRUPPER_NAY],
    [Stønadstype.LÆREMIDLER]: [...MÅLGRUPPER_NAY],
    [Stønadstype.BOUTGIFTER]: [...MÅLGRUPPER_NAY],
    [Stønadstype.DAGLIG_REISE_TSO]: [...MÅLGRUPPER_NAY],
    [Stønadstype.DAGLIG_REISE_TSR]: [FaktiskMålgruppe.TODO],
};
```